bundles/org.openhab.binding.benqprojector/src/main/resources/OH-INF/i18n/benqprojector.properties

```diff
@@ -82,3 +81,18 @@ channel-type.benqprojector.source.state.option.rgb = Computer/YPbPr
 channel-type.benqprojector.source.state.option.rgb2 = Computer/YPbPr2
 channel-type.benqprojector.source.state.option.vid = Video
 channel-type.benqprojector.source.state.option.svid = S-Video
+
+# thing types
+
+thing-type.benqprojector.discovery.label = BenQ Projector
+
+# channel types
+
+channel-type.benqprojector.aspectratio.state.option.4:3 = 4:3
+channel-type.benqprojector.aspectratio.state.option.16:9 = 16:9
+channel-type.benqprojector.directcmd.state.option.mute = on = Mute On
+channel-type.benqprojector.directcmd.state.option.mute = off = Mute Off
+channel-type.benqprojector.directcmd.state.option.vol = + = Volume +
+channel-type.benqprojector.directcmd.state.option.vol = - = Volume -
+channel-type.benqprojector.directcmd.state.option.menu = on = Menu On
+channel-type.benqprojector.directcmd.state.option.menu = off = Menu Off
```

Comment:
These don't look right.

Reply:
True. 
I will remove those additions, please note the escaping of the equals sign above. Seems like a bug in the i18n add-on?